ext/mbstring/mbstring.c

```diff
@@ -2933,6 +2933,10 @@ PHP_FUNCTION(mb_convert_kana)
 
 	string.val = (unsigned char*)string_val;
 
+#define DISALLOW_FLAG(flag, opt1, opt2) do { if (opt & flag) { \
+	zend_argument_value_error(2, "must not combine '%c' and '%c' flags", opt1, opt2); \
+	RETURN_THROWS(); }} while(0)
```

Comment:
Please break this up into multiple lines.

Reply:
Done.